integration_tests/integration/barcode/pdfCoversheet.spec.ts

```diff
@@ -0,0 +1,23 @@
+import Page from '../../pages/page'
+import PrintCoversheetsPage from '../../pages/barcode/printCoversheets'
+import FindRecipientByPrisonNumberPage from '../../pages/barcode/findRecipientByPrisonNumber'
+
+context('Print PDF coversheet', () => {
+  let printCoversheetsPage: PrintCoversheetsPage
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubCreateBarcode')
+    printCoversheetsPage = PrintCoversheetsPage.goToPage()
+  })
+
+  it('should allow the PDF to be downloaded again', () => {
```

Comment:
I was desperately trying to write methods in the page class to assert the file had been downloaded, but 'varying degrees of success' is the polite way of saying how far I got with it!
The code was getting increasingly hacky and indeterminate, and in the end I decided it wasnt worth the effort